File: src/test/programmers/level2/CreateJadenCaseString.java
package test.programmers.level2;

public class CreateJadenCaseString {
    public static void main(String[] args) {
        System.out.println(solution("3people unFollowed me"));
    }

    static String solution(String s) {
        String answer = "";
        int idx = 0;
        String[] str = s.toLowerCase().split("");
        StringBuilder sb = new StringBuilder();

        for(int i = 0; i < str.length; i++) {
            if(idx == 0) {
                sb.append(str[i].toUpperCase());
            } else {
                sb.append(str[i].toLowerCase());
            }
            idx++;

            if(str[i].equals(" ")) {
                idx = 0;
            }
        }

        answer = sb.toString();

        return answer;
    }
}

File: src/test/programmers/python/level2/CreateJadenCaseString.py
def solution(s):
    result = ''
    isFirst = True
    for i in range(len(s)):
        if isFirst:
            result += s[i].upper()
            isFirst = False
        else:
            result += s[i].lower()

        if s[i] == " ":
            result += ""
            isFirst = True

    return result

# def solution(s):
#     item = s.split(" ")
#     result = []
#     for i in item:
#         result.append(i[0].upper() + i[1:].lower())
#     return " ".join(result)

print(solution("3people unFollowed me"))
print(solution("for the last week"))